src/ensembl/src/shared/components/help-popup/HelpPopupBody.scss

```diff
@@ -70,6 +80,11 @@ $aside-left-padding: 1.6rem;
 .relatedVideo {
   color: $blue;
   cursor: pointer;
+  margin-bottom: 0.4rem;
+}
+
+.relatedCurrent {
+  color: inherit;
```

Comment:
Do we need to reset the cursor to `default` here?
![help cursor](https://user-images.githubusercontent.com/3085815/97295717-97980880-1847-11eb-91a2-ee72c94e895f.gif)

Reply:
Yes. Updated the styles.